pkg/scannerv4/mappers/mappers.go

```diff
@@ -386,10 +373,71 @@ func toProtoV4VulnerabilitiesMap(ctx context.Context, vulns map[string]*claircor
 			FixedInVersion:     fixedInVersion(v),
 			Cvss:               cvss,
 		}
+		var cvssMetrics []*v4.VulnerabilityReport_Vulnerability_CVSS
+		if rhelSev, err := rhelSeverityAndScores(v); err == nil {
```

Comment:
calling this + `nvdSeverityAndScores` and `cvssVector` seems pretty redundant, as they are already called earlier in `severityAndScores`. Perhaps we should just update `severityAndScores` to handle this logic, instead, so we don't have to do it again here

Reply:
Yeah, I agree. I thought the `CVSS` field was retiring so eventually we will get rid of this line `sev, err := severityAndScores(ctx, v, &nvdVuln)`.
I will adjust the logics.